VideoLocker/src/test/java/org/edx/mobile/view/CourseUnitNavigationActivityTest.java

```diff
@@ -397,6 +395,7 @@ public void testUnitFragmentCreation() {
         CourseUnitFragment.HasComponent hasComponent = Mockito.mock(CourseUnitFragment.HasComponent.class);
 
         List<CourseComponent> unitList = new ArrayList<>();
+        List<Class> classesList = new ArrayList<>();
```

Comment:
Note that you can use a single `List<Pair<CourseComponent, Class<? extends CourseUnitFragment >>>` to provide an implicit guarantee that the two lists are equal and correspond with each other.